src/app/core/teams/teams.controller.ts

```diff
@@ -1,16 +1,15 @@
-'use strict';
+import { dbs, utilService, auditService } from '../../../dependencies';
+import UserService from '../user/user.service';
+import { TeamRoles } from './team-role.model';
+import TeamsService from './teams.service';
```

Comment:
Can we revert `UserService` and `TeamsService` back to `userService` and `teamService`?  This will be consistent with how they are imported elsewhere as well as prevent confusion between the instance and the class.